Asymmetrik/ansible-roles
# FH-3987 Diode Software Artifacts

**Acceptance Criteria**

- Diode Software Artifacts pipeline is established
- Allowed users are able to upload an artifact to firehawk-diode-yard and receive the artifact away within 30 minutes.

**Change Description**

The Ansible role will setup diode manifest generation automatically in mgmt-diode-signer. The Jenkins account is currently being used, but this can probably be switched to root if wanted.

The Terraform code relies on the module in https://bitbucket.org/asymmetrik/terraform-modules/pull-requests/213/fh-3987-diode-software-artifacts. This has already been synced to the firehawk-master branch in terraform-modules.

**Verification Instructions**

Upload a file to firehawk-diode-yard. Wait 30 minutes. Check away to see if it has arrived in Diode.

**Background, Context & Notes**

This is ParkPlace's replacement to Portkey. This Ansible role is used with PRs https://bitbucket.org/asymmetrik/terraform-modules/pull-requests/213 and https://bitbucket.org/asymmetrik/firehawk-deploy/pull-requests/697.

Git syncing is not part of this PR - it will be handled in another ticket since it would require another process to auto-generate Git packages home-side and processes away to automatically pick up + sync those Git packages to the repo away.
